tests/evm/test_extcodehash.py

```diff
@@ -48,11 +65,24 @@ def test_extcodehash(address: U160, nonce: U256, balance: U256, code: bytes, is_
     tx_id = 1
     call_id = 1
 
+    rw_counter_end_of_reversion = 0 if is_persistent else 9
+    state_write_counter = 0
+
     rw_table = set(
         RWDictionary(0)
```

Comment:
```suggestion
        RWDictionary(1)
```
rw counter should start with 1